cypress/e2e/remove-liquidity.test.ts

```diff
@@ -1,7 +1,35 @@
+import { ChainId, MaxUint256 } from '@uniswap/sdk-core'
+
+import { UNI } from '../../src/constants/tokens'
+
+const UNI_MAINNET = UNI[ChainId.MAINNET]
```

Comment:
nit: you're pulling in the token here but you can just pull in the address directly by using `UNI_ADDRESSES[ChainId.MAINNET]` and importing `UNI_ADDRESSES from '@uniswap/sdk-core'`